### PR TITLE
Parse Symbols

### PR DESCRIPTION
**Pull request by b4l**

### CODE REVIEW

File: src/MapfileStyleParser.ts [hunk 1]
```diff
@@ -314,24 +320,100 @@ export class MapfileStyleParser implements StyleParser {
...
     return markSymbolizer;
   }
 
+  /**
+   * Get the GeoStyler-Style IconSymbolizer from an Mapfile Style
+   *
+   * @param {object} styleParameters The Mapfile Style
+   * @return {IconSymbolizer} The GeoStyler-Style IconSymbolizer
+   */
+  getIconSymbolizerFromMapfileStyle(styleParameters: any): IconSymbolizer {
```
Comment (ger-benjamin) [inline src/MapfileStyleParser.ts:351]: Can you merge some lines in a common function for getIconSymbolizerFromMapfileStyle and getTextSymbolizerFromMapfileStyle ?

Comment (b4l) [reply src/MapfileStyleParser.ts:351]: Done in #10

**Comment by b4l** [comment]
Moved the symbols.sym and rebased.

**Comment by ger-benjamin** [comment]
Thanks Please merge, I'll add later symbol to the container for mapserver (or do it in your next PR)

**Pull request merged**